content-api-first-responder.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=first-responder
+
+[Service]
+User=content-api
+Group=content-api
+Restart=on-failure
+Environment='HOME=/home/content-api'
+Environment='JAVA_OPTS=-XX:+UseConcMarkSweepGC -XX:NewRatio=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:/home/content-api/logs/gc.log -XX:ReservedCodeCacheSize=256m -Dconfig.file=/etc/gu/content-api-first-responder.conf'
```

Comment:
Probably want to set an adequate heap size (http://crunchify.com/jvm-tuning-heapsize-stacksize-garbage-collection-fundamental/) for the JVM given the size of your instance.